src/test/java/org/mifos/integrationtest/ClientCorrelationIdTest.java

```diff
@@ -0,0 +1,91 @@
+package org.mifos.integrationtest;
+
+import com.google.gson.Gson;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mifos.integrationtest.common.CollectionHelper;
+import org.mifos.integrationtest.common.Utils;
+import org.mifos.integrationtest.common.dto.CollectionResponse;
+import org.mifos.integrationtest.common.dto.OperationsHelper;
+import org.mifos.integrationtest.common.dto.operationsapp.GetTransactionRequestResponse;
+import org.mifos.integrationtest.common.dto.operationsapp.TransactionRequest;
+import org.mifos.integrationtest.config.ChannelConnectorConfig;
+import org.mifos.integrationtest.config.OperationsAppConfig;
+import org.mifos.integrationtest.cucumber.BaseStepDef;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static com.google.common.truth.Truth.assertThat;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ClientCorrelationIdTest {
+
+    private ResponseSpecification statusOkResponseSpec;
+    private RequestSpecification requestSpec;
+
+    private String transactionId = "fce838977c90oKNEILYY";
+    private String clientCorrelationId = "123456789";
+
+    @BeforeAll
+    public void setup() {
+        this.requestSpec = new RequestSpecBuilder().setContentType(ContentType.JSON).build();
+        this.requestSpec.header(Utils.TENANT_PARAM_NAME, "gorilla");
```

Comment:
cant we take this as a configuration? It's not a good idea to hardcode values in code